### PR TITLE
remove worldmap/geoconnect #7409

### LINKED ISSUE #7409: Remove Worldmap/Geoconnect from Application
Following Harvard Worldmap's migration to new infrastructure, we will no longer be able to apply resources to reconfigure and support Geoconnect, which powers the integration. :/ Since we're actively investigating other geospatial exploration and analysis options in the community and with Harvard Library, I'm going to suggest pulling out Worldmap from the application. We'll need to pull it out of the guides and automated tests as well. This integration was built before we had external tools, and any new geospatial exploration support should follow our modular framework.

### PR DESCRIPTION
**Pull request by pdurbin**
**What this PR does / why we need it**:

As explained in #7409 we are removing WorldMap/Geoconnect support from Dataverse.

**Which issue(s) this PR closes**:

Closes #7409

**Special notes for your reviewer**:

The issue says to grab a backup but I took this to mean a full database backup. I didn't implement anything special.

**Suggestions on how to test this**:

Poke around in the area where the Map It (Geoconnect) button used to be as well as the area where you could view maps.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

Yes, maps are gone. The link to create maps is gone.

**Is there a release notes update needed for this change?**:

Probably a good idea but I didn't add one yet.

**Additional documentation**:

None.

### CODE REVIEW

**Comment by djbrooke** [comment]
Thanks @pdurbin - if someone wants to start reviewing the code that would be great. I'll take a look at docs and add release notes later this afternoon.

**Comment by djbrooke** [comment]
Ok - release notes added and docs reviewed. Looks good! Still needs some code review.

**Review by qqmyers — APPROVED** [review]
While this is large, it looks like everything is a ~straight-forward deletion of Worldmap code and/or simplifying logic in conditional statements to drop the Worldmap special case. 
I didn't check every line, but spot checking all seems good and due to the nature of the change, I'd think moving this on to QA would make sense. The things that could break would presumably be around the external tools displays.

File: src/main/java/edu/harvard/iq/dataverse/api/Access.java [hunk 1]
```diff
@@ -1794,7 +1765,7 @@ private boolean isAccessAuthorized(DataFile df, String apiToken) {
                 return false;
             } 
             
-        if (permissionService.requestOn(createDataverseRequest(user), df).has(Permission.DownloadFile)) { 
+        if (permissionService.requestOn(createDataverseRequest(user), df).has(Permission.DownloadFile)) { // FIXME: Indent this line more.
```
Comment (qqmyers) [inline src/main/java/edu/harvard/iq/dataverse/api/Access.java:1768]: Can this just be changed? (was this a test to see if reviewers would notice :-) ? )

Comment (pdurbin) [reply src/main/java/edu/harvard/iq/dataverse/api/Access.java]: Fixed in a71841346. I wouldn't say it was a test. I always try to keep my diffs as small as possible. 😄

**Pull request merged**